docs/security-champion/7-about.md

```diff
@@ -0,0 +1,29 @@
+# About the network
+
+The Security Champion Network is intended to be a community for Security Champions in Equinor. Software development over the last years has rapidly evolved from big development teams consisting of dozens of developers to smaller autonomous teams where we are today. With greater responsibility of the whole lifecycle of applications, modern DevOps teams are also expected to handle security.
+
+This network rises from the realization that this shift-left thinking when it comes to security requires competence building within the developer teams. As the network grows we strive for it to become a powerhouse in Equinor when it comes to application security. It will be a place where teams can exchange experiences - _both good and bad_, and hopefully learn from each others.
+
+## Desired outcome
+
+Create a lively community for people working with Application Security in Equinor. Knowledge is shared across teams, and we are then able to scale security in a more impactful way.
+
+## Activities
+
+Please check our [event](./events/index.md) site for info about upcoming and past Security Champion network events.
+
+The main communication channel for the community is [Slack](https://app.slack.com/client/T02JL00JU/CMM6FSW5V). This is where people can post questions exchange experiences when it comes to different tools and technologies etc.
+
+## Responsibilities
+
+Everyone in the community is responsible for keeping these info sites up to date. They are all based on markdown, don't hesitate to make a PR with fixes, or some new guidelines!
+
+## Role of the AppSec team
+
+The AppSec team is responsible for driving this network, but this does not mean that you can't contribute!
+
+Would you like to take on some extra responsibilities when it comes to organizing events and keeping there pages up to date? Please reach out to us on [Slack](https://app.slack.com/client/T02JL00JU/CMM6FSW5V) or e-mail at ``appsec[at]equinor.com``
```

Comment:
Possibly misleading. Reading this paragraph as a whole it sounds like, as a security champion, I either let the AppSec team do all the work, or I need to officially sign up (and it is not clear what signing up implies). Can we rephrase this to make it "easier" for people to contribute? 
You have a good idea? Go for it! You want to go big? Get in touch! 🤟